internal/bminventory/inventory_test.go

```diff
@@ -18892,3 +18882,28 @@ var _ = Describe("V2UpdateHostIgnition unbound blabla", func() {
 		Expect(response).To(BeAssignableToTypeOf(&installer.V2UpdateHostIgnitionCreated{}))
 	})
 })
+
+func getDummyCluster() common.Cluster {
+	clusterID := strfmt.UUID(uuid.New().String())
+	c := common.Cluster{
+		Cluster: models.Cluster{
+			ID:            &clusterID,
+			Name:          "my-cluster",
+			BaseDNSDomain: "my-domain",
+		},
+	}
+	return c
+}
+
+func createClusterWithMonitoredOperator(db *gorm.DB, operator models.MonitoredOperator) common.Cluster {
+	c := getDummyCluster()
+	c.MonitoredOperators = append(c.MonitoredOperators, &operator)
+	Expect(db.Create(&c).Error).ShouldNot(HaveOccurred())
+	return c
+}
+
+func createClusterWithoutMonitoredOperator(db *gorm.DB) common.Cluster {
```

Comment:
Nit: Isn't this just a createCluster function?
If so, can we just reuse this one https://github.com/openshift/assisted-service/blob/9f84d8570761e0b4f4fe3b6bfdafbfabccb62243/internal/bminventory/inventory_test.go#L390

Reply:
Nice, well spotted. I'll move it and reuse the function you mention